sql/src/main/java/io/crate/planner/node/dql/CollectNode.java

```diff
@@ -157,6 +158,14 @@ public void jobId(@Nullable UUID jobId) {
         this.jobId = Optional.fromNullable(jobId);
     }
 
+    public Optional<List<String>> ignoreUnsupportedFields() {
+        return ignoreUnsupportedFields;
+    }
+
+    public void ignoreUnsupportedFields(@Nullable List<String> ignoreUnsupportedFields) {
```

Comment:
I am not sure if this should really be part of the CollectNode or configurable at all.

Since _version is prohibited in the Planner for all cases where it doesn't work maybe we should just hardcode the LuceneQueryBuilder to always ignore _version ?